pkg/streamingpromql/planning/plan.proto

```diff
@@ -20,7 +20,7 @@ message EncodedQueryPlan {
 
   bool enableDelayedNameRemoval = 5;
 
-  int64 version = 6;
+  uint64 version = 6;
```

Comment:
Note to reviewers: this change is considered safe given the only values we set this to at present are 0 or 1.

Docs: https://protobuf.dev/programming-guides/proto3/#conditionally-safe-changes